policygenerator/policy-sets/community/openshift-plus-setup/machine-sets.yaml

```
@@ -149,7 +149,12 @@ spec:
                   numCPUs: '{{ fromConfigMap "policies" "opp-settings" "vmwareCPU" | toInt }}'
                   numCoresPerSocket: 4
                   snapshot: ""
-                  template: {{ (lookup "config.openshift.io/v1" "Infrastructure" "" "cluster").status.infrastructureName }}-rhcos-generated-region-generated-zone
+                  {{- $tmpsuffix := "region-generated-zone" }}
+                  {{- $ocpversion := semver (fromClusterClaim "version.openshift.io") }}
+                  {{- if gt $ocpversion.Minor 18 }}
+                    {{- $tmpsuffix = "failure-domain" }}
+                  {{- end }}
+                  template: {{ (lookup "config.openshift.io/v1" "Infrastructure" "" "cluster").status.infrastructureName }}-rhcos-generated-{{ $tmpsuffix}}
```

Shouldn't the hyphen be included inside of the variable `tmpsuffix`?
```suggestion
                  template: {{ (lookup "config.openshift.io/v1" "Infrastructure" "" "cluster").status.infrastructureName }}-rhcos-generated{{ $tmpsuffix}}
```

---

there's always a suffix, but I can move it

---

Oh, no need to move it--it's better as it is. I didn't notice it was initialized with a different value. Thanks!